src/App.tsx

```diff
@@ -1,24 +1,114 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Select } from './types/Select';
+import { getTodos } from './api/todos';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+import { Errors } from './types/Errors';
+import { Header } from './components/Header';
 
-const USER_ID = 0;
+export const USER_ID = 9960;
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[] | null>(null);
+  const [selectedFilter, setSelectedFilter] = useState(Select.ALL);
+  const [typeError, setTypeError] = useState<Errors | null>(null);
+  const [notificationError, setNotificationError] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [loadersTodosId, setLoadersTodoId] = useState<number[] | null>(null);
```

Comment:
in the case when array is stored in the state set empty array as the default value
```suggestion
  const [loadingTodoIds, setLoadingTodoIds] = useState<number[] | null>([]);
```
also, `set` function name should always match the state variable name